src/index.js

```diff
@@ -1 +1,83 @@
 import './style.css';
+import Shows from './shows.js';
+import { showsContainer, searchForm, searchFormInput } from './dynamic.js';
+import likes from './images/Empty_Like.svg';
+
+const setNewShow = (programList) => {
+  let tvShows = [];
+  programList.forEach((program) => {
+    const { id, image } = program.show;
+    const title = program.show.name;
+    const like = false;
+    const newShow = new Shows(id, image, title, like);
+    if (image) {
+      tvShows = [...tvShows, newShow];
+    }
+  });
+  return tvShows;
+};
+
```

Comment:
[optional] Kindly avoid having different functions in a single js file as they make the js file appear clumsy. You can break down the code depending on different functionalities or features